kerchunk/intro.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Kerchunk"
+subtitle: References for (optionally) chunked, compressed n-dimensional arrays
+---
+
+## What is Kerchunk?
+
+Kerchunk is a python library for creating reference files to support cloud-optimized access to traditional geospatial file formats, like NetCDF. Kerchunk negates the need to create and store copies of data for cloud-optimized access. Given the challenge of creating and maintaining copies of data, Kerchunk is a great tool.
+
+::: {.callout-warning}
+A major limitation of kerchunk is the chunking of data will always be constrained to the chunk structure of the underlying data format.
+:::
+
+Kerchunk creates [zarr](../zarr/intro.qmd) metadata, stored in a "reference file", for non-zarr data. Kerchunk enables a unified way to access chunked, compressed n-dimensionsional data across a variety of conventional data formats. The kerchunk library now supports NetCDF/HDF5, GRIB2, TIFF.
+
+Learn more about kerchunk at [kerchunk.readthedocs.io](https://kerchunk.readthedocs.io/en/latest/).
+
+## How to kerchunk 
+
+[`kerchunk`](https://fsspec.github.io/kerchunk) is a python library you can use to create a reference file from any of the file formats it supports. At time of writing, Kerchunk can create reference files for NetCDF4/HDF5, GRIB2, FITS, TIFF and NetCDF3. Check the [File format backends](https://fsspec.github.io/kerchunk/reference.html#file-format-backends) section of the kerchunk documentation for updates to supported formats.
+
+The reference file is used by the `fsspec.ReferenceFileSystem` to read data from local or remote storage.
+
+Here's a simple example:
```

Comment:
remove the term `simple`, just say `an example` (Data/Software Carpentries best practices)